workspaces/airbrake/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": "18 || 20"
+    "node": "18 || 20 || 22"
```

Comment:
For all the plugins we own as Community Plugin Maintainers I think we should follow the upstream policy and only support the latest LTS versions so that would be 20 and 22.

Reply:
Addressed this in [3fc8cf1](https://github.com/backstage/community-plugins/pull/3562/commits/3fc8cf19007c5f1bfe058d7bb353be3e4d2df8da) by updating maintainer-owned workspaces.